bin/ChakraCore/CMakeLists.txt

```diff
@@ -3,10 +3,6 @@ add_library (ChakraCore SHARED
     ConfigParserExternals.cpp
     TestHooks.cpp
 )
-
-# By default, don't export any symbols from this library
-# We'll manually export the relevant individual functions
-set_target_properties(ChakraCore PROPERTIES CXX_VISIBILITY_PRESET hidden)
```

Comment:
This is a false setting..So, it doesn't affect statically inherited libraries, hence linker gives warnings.